changelog

```diff
@@ -1,3 +1,20 @@
+Version 0.1.2
```

Comment:
Please add a link to the msal wiki at the top of the release: https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki